src/pydata_sphinx_theme/assets/styles/base/_base.scss

```diff
@@ -59,6 +59,21 @@ a {
       opacity: 1;
     }
   }
+
+  // set up a icon next to the shorten links from github and gitlab
+  &::before {
+    color: var(--pst-color-text-muted);
+    font-family: "Font Awesome 5 Brands";
```

Comment:
Note that this will need to be 6 once we finish #881

Reply:
done